src/core/auth/atoms/userWasLanded.ts

```diff
@@ -1,25 +1,11 @@
 import { configRepo } from '~core/config';
-import { createAtom } from '~utils/atoms';
 import { localStorage } from '~utils/storage';
 
-export const userWasLandedAtom = createAtom(
-  {},
-  ({ get }, state: 'yes' | 'no' | 'pending' = 'pending') => {
-    const userLoaded = true;
-    if (userLoaded) {
-      const initialUrl = new URL(localStorage.getItem('initialUrl') || '');
-      const noSettingUrl =
-        location.pathname === configRepo.get().baseUrl && initialUrl.search === '';
-      const userHaveLandedMark = localStorage.getItem('landed');
-      if (noSettingUrl && !userHaveLandedMark) {
-        state = 'no';
-      } else {
-        state = 'yes';
-      }
-    }
-    return state;
-  },
-  'userWasLandedAtom',
-);
-
 export const landUser = () => localStorage.setItem('landed', 'true');
+export function userWasLanded() {
+  const initialUrl = new URL(localStorage.getItem('initialUrl') || '');
+  const noSettingUrl =
```

Comment:
Using negation in variable names complicates code comprehension. Better to avoid it